src/org/zaproxy/zap/extension/ascanrulesAlpha/ZapAddOn.xml

```diff
@@ -9,7 +9,7 @@
 	<![CDATA[
 	Add .htaccess scanner (Issue 3972).<br>
 	Modified trace.axd scanner to leverage new AbstractAppFilePlugin component.<br>
-	]]>
+	]]><![CDATA[Add NoSQL scan rule(s) (Issue 3480).<br>Ad a new Tech variable (into the Tech class) for MongoDB.<br>Added some entry int the Messages.properties  file. ]]>

```

Comment:
This line should be part of the existing CDATA block. 
Only the first sentence is required here (the core changes aren't applicable to the extension, and the message changes aren't relevant to the end user).